v1/types/types.gunk

```diff
@@ -252,6 +252,26 @@ type Location struct {
 	Longitude float64 `pb:"2" json:"longitude"`
 }
 
+type OfflineUserInfo struct {
+	// UserID
+	UserID string `pb:"1" json:"user_id"`
+
+	// FirstName of the person
+	FirstName string `pb:"2" json:"first_name"`
+
+	// MiddleName `pb:"" json:""` or middle names (space separated) of the person
```

Comment:
I think `pb:"" json:""` annotation here is a typo? Same goes to MiddleName, LastName, and MobileNo fields?

Reply:
Fixed.